### PR TITLE
[DOC] Add import k8s context

### LINKED ISSUE #4163: Create a doc that describes importing kubernetes contexts
based on https://github.com/pachyderm/pachyderm/pull/4152

### PR DESCRIPTION
**Pull request by svekars**
Closes #4163

### CODE REVIEW

File: doc/deployment/import-kubernetes-context.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+1. Create a new Pachyderm context with the embedded Kubernetes context:
+
+   ```bash
+   $ pachctl config set context <new-pachyderm-context> -k `kubectl config current-context`
+   ```
+
+   **Example:**
+
+   ```bash
+   $ pachctl config set context test-context -k `kubectl config current-context`
```
Comment (JoeyZwicker) [inline doc/deployment/import-kubernetes-context.md]: I dont think you need this example and the above. 

pachctl config set context <new-pachyderm-context> -k `kubectl config current-context` is pretty c;ear without needing to show an example where you replace one arugment

File: doc/deployment/import-kubernetes-context.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+1. Verify that the new context has been activated:
+
+   ```bash
+   $ pachctl config get active-context
+   ```
+
+   **Example:**
+
+   ```bash
+   $ pachctl config get active-context
```
Comment (JoeyZwicker) [inline doc/deployment/import-kubernetes-context.md]: same as above, we don't need this pattern of showing a command AND showing an example that is exactly or nearly the cmd but with output. just one is fine. We only need to include both when there are multiple arguments that might not be clear or it's a cmd that we expect the user to copy/paste and use as part of a tutorial

File: doc/deployment/import-kubernetes-context.md (hunk 1 above)
Comment (svekars) [reply doc/deployment/import-kubernetes-context.md]: I can remove them, but overall they don't hurt, IMO.

File: doc/deployment/import-kubernetes-context.md (hunk 2 above)
Comment (svekars) [reply doc/deployment/import-kubernetes-context.md]: removed

File: doc/deployment/import-kubernetes-context.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+1. Verify that the context was successfully created and view the context parameters:
+
+   **Example:**
+
+   ```bash
+   $ pachctl config get context test-context
+   {
+     "source": "IMPORTED",
+     "cluster_name": "minikube",
+     "auth_info": "minikube"
```
Comment (pappasilenus) [inline doc/deployment/import-kubernetes-context.md]: I believe this should also have this line:

"namespace": "default",

When https://github.com/pachyderm/pachyderm/pull/4170 is merged.

Until then, maybe the doc should use the workaround, and tell folks to specify namespaces?

https://github.com/pachyderm/pachyderm/issues/4166#issuecomment-538437301

Comment (svekars) [reply doc/deployment/import-kubernetes-context.md]: It looks like the #4170 will be merged shortly. I'll wait for that to happen and will add the namespace field once that happens. I don't think it's worth it to add a workaround only to change it in a day or so.

**Comment by svekars** [comment]
Since #4170 was merged, updating accordingly and it should be ready to merge.

**Pull request merged**